app/models/listing.js

```diff
@@ -0,0 +1 @@
+export { default } from '@lblod/ember-submission-form-fields/models/listing';
```

Comment:
Nitpick, can be done in the future since this pattern was already used before: 

I don't think the "models" need to be reexported from the app folder. These are only used internally by the addon I think? And even if apps would use them directly they would probably still import them from `@lblod/embersubmission-form-fields/models/listing.js` instead of `app-name/models/listing.js`?